packages/riipen-ui/src/components/Tooltip.jsx

```diff
@@ -0,0 +1,427 @@
+import clsx from "clsx";
+import PropTypes from "prop-types";
+import React from "react";
+import css from "styled-jsx/css";
+
+import ThemeContext from "../styles/ThemeContext";
+
+import withClasses from "../utils/withClasses";
+
+import Popover from "./Popover";
+
+class Tooltip extends React.Component {
+  static propTypes = {
+    /**
+     * The content to trigger the tooltip with.
+     */
+    children: PropTypes.node,
+
+    /**
+     * Any additional classes to apply.
+     */
+    classes: PropTypes.array,
+
+    /**
+     * Color of tooltip.
+     */
+    color: PropTypes.oneOf([
+      "default",
+      "white",
+      "positive",
+      "negative",
+      "warning"
+    ]),
+
+    /**
+     * Whether tooltip should display on click.
+     */
+    click: PropTypes.bool,
+
+    /**
+     * The component used for the root node.
+     * Either a string to use a DOM element or a component.
+     */
+    component: PropTypes.elementType,
+
+    /**
+     * The location to attach the anchor to on the content element.
+     * Also used to determine the tooltip arrow placement/direction.
+     */
+    contentPosition: PropTypes.object,
```

Comment:
I would consider changing this to be just position with values ["above", "below", "left", "right"] and the calculate the anchor and content position based on that. I dont think we need that much extensibility and it would simplify the API considerably. Thoughts?

Reply:
That does sound nice!  Would we be able to combine the positions to have things like top-right, bottom-left etc?  

I have a use case in design where I want the arrow in the top left corner, which is why I think I went with contentPosition, but happy to update!

Reply:
yeah, we could do the 8 positions and I would suggest setting the anchor and content positions accordingly so that they line up properly.
ie: 
// above-left
anchorPosition: { vertical: "top", horizontal: "left" }
contentPosition: { vertical: "bottom", horizontal: "right" }

// below
anchorPosition: { vertical: "bottom", horizontal: "center" }
contentPosition: { vertical: "top", horizontal: "center" }

Reply:
Added the positions prop, used top/bottom instead of above/below because it was easy to parse 🙈 Let me know what you think!